packages/components/src/components/primitives/Button/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { Variant } from './types';
+
+export const BUTTON_VARIANTS: Record<string, Variant> = {
```

Comment:
why do you need this?

Reply:
we might use around the whole application to use the variant instead of hardcode the values primary or secondary when can use the constant.

Reply:
chakra provides code completion for typescript, it will actually _work better_ without this

Reply:
moreover, this is not consistent with other components, please [take a look](https://github.com/cmpsr/composer/blob/with-chakra/packages/components/src/components/primitives/TextPairing/types.ts) at how we implemented this for the TextPairing component

Reply:
I decided to remove this file and use the definitions types as it was used in the TextPairing implementation